src/styles/cards/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $border-color-card;
+  justify-content: center;
+  display: block;
+  flex-direction: column;
+  padding: 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    padding: 16px 4px 30px;
+    display: flex;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: $line-height;
+    font-weight: 500;
+    color: $color-title;
+    margin-bottom: -6px;
+  }
+
+  &__title:hover {
+    color: #34568b; // Правило для ховера на заголовок отдельно
```

Comment:
The color for the hover state of your title should ideally be stored in a variable for consistency and easy management. Consider moving '#34568b' to a variable in your '_variables.scss' file.